Testscripts/Linux/perf_memcached.sh

```diff
@@ -0,0 +1,231 @@
+#!/bin/bash
+
+#######################################################################
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+#######################################################################
+
+#######################################################################
+#
+# perf_memcached.sh
+# Description:
+#    Install memcached and run memcached performance tests.
+#    This script needs to be run on client VM.
+#
+# Supported Distros:
+#    Ubuntu/Suse/Centos/RedHat/Debian
+#######################################################################
+
+CONSTANTS_FILE="./constants.sh"
+UTIL_FILE="./utils.sh"
+
+PORT="21789"
+THREADS=(1 2 4 8 16 32 64 128 256 512)
+max_threads=16
```

Comment:
Some comments as why 16

Reply:
This is also inherited from lis-test repo.